gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var gulp = require('gulp');
+var jeditor = require('gulp-json-editor');
+var merge = require('merge-stream');
+var zip = require('gulp-zip');
+
+var files = ['background.js', 'contentScript.js', 'logo.png',
```

Comment:
This is not my repo, but a small suggestion: if you move all these files under a directory (maybe `src`?), then you can just use a globbing pattern to grab all files, which will be a bit more future-proof.